<!DOCTYPE html>
<html>

  <head>
 <meta charset="utf-8">
<title>Interactive Visual Exploration And Refinement Of Cluster Assignments</title>
<meta name="viewport" content="width=device-width">
<meta name="description" content="Caleydo: Visualization for BioMedicine.">
<link rel="canonical" href="http://localhost:4000/publications/2017_bmc_clustering/">

<!-- Custom Fonts -->
<link href='//fonts.googleapis.com/css?family=Lato:300,400,700,900,300italic,400italic,700italic,900italic|Patua+One|Vollkorn:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
<link href='//fonts.googleapis.com/css?family=PT+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>

<!-- Custom CSS -->
<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
<link rel="stylesheet" href="/css/site.css">

<!-- Fav and touch icons -->
<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
<link rel="apple-touch-icon" href="/favicon.png" />

 <!-- metadata for link preview  -->
  <!-- Schema.org markup for Google+ -->
  <meta itemprop="name" content="Interactive Visual Exploration And Refinement Of Cluster Assignments">
  <meta itemprop="description" content=" Background: With ever-increasing amounts of data produced in biology research, scientists are in need of efficient data analysis methods. Cluster analysis, combined with visualization of the results, is one such method that can be used to make sense of large data volumes. At the same time, cluster analysis is known to be imperfect and depends on the choice of algorithms, parameters, and distance measures. Most clustering algorithms don't properly account for ambiguity in the source data, as records are often assigned to discrete clusters, even if an assignment is unclear. While there are metrics and visualization techniques that allow analysts to compare clusterings or to judge cluster quality, there is no comprehensive method that allows analysts to evaluate, compare, and refine cluster assignments based on the source data, derived scores, and contextual data.  Results: In this paper, we introduce a method that explicitly visualizes the quality of cluster assignments, allows comparisons of clustering results and enables analysts to manually curate and refine cluster assignments. Our methods are applicable to matrix data clustered with partitional, hierarchical, and fuzzy clustering algorithms. Furthermore, we enable analysts to explore clustering results in context of other data, for example, to observe whether a clustering of genomic data results in a meaningful differentiation in phenotypes. Conclusions: Our methods are integrated into Caleydo StratomeX, a popular, web-based, disease subtype analysis tool. We show in a usage scenario that our approach can reveal ambiguities in cluster assignments and produce improved clusterings that better differentiate genotypes and phenotypes.">
  
  <meta itemprop="image" content="http://localhost:4000/assets/images/papers/2017_bmc_clustering_teaser.png">
  

  <!-- Twitter Card data -->
  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@caleydo_org">
  <meta name="twitter:title" content="Interactive Visual Exploration And Refinement Of Cluster Assignments">
  <meta name="twitter:description" content=" Background: With ever-increasing amounts of data produced in biology research, scientists are in need of efficient data analysis methods. Cluster analysis, combined with visualization of the results, is one such method that can be used to make sense of large data volumes. At the same time, cluster analysis is known to be imperfect and depends on the choice of algorithms, parameters, and distance measures. Most clustering algorithms don't properly account for ambiguity in the source data, as records are often assigned to discrete clusters, even if an assignment is unclear. While there are metrics and visualization techniques that allow analysts to compare clusterings or to judge cluster quality, there is no comprehensive method that allows analysts to evaluate, compare, and refine cluster assignments based on the source data, derived scores, and contextual data.  Results: In this paper, we introduce a method that explicitly visualizes the quality of cluster assignments, allows comparisons of clustering results and enables analysts to manually curate and refine cluster assignments. Our methods are applicable to matrix data clustered with partitional, hierarchical, and fuzzy clustering algorithms. Furthermore, we enable analysts to explore clustering results in context of other data, for example, to observe whether a clustering of genomic data results in a meaningful differentiation in phenotypes. Conclusions: Our methods are integrated into Caleydo StratomeX, a popular, web-based, disease subtype analysis tool. We show in a usage scenario that our approach can reveal ambiguities in cluster assignments and produce improved clusterings that better differentiate genotypes and phenotypes.">
  <meta name="twitter:creator" content="@caleydo_org">
  
  <meta name="twitter:image:src" content="http://localhost:4000/assets/images/papers/2017_bmc_clustering_teaser.png">
  

  <!-- Open Graph data -->
  <meta property="og:title" content="Interactive Visual Exploration And Refinement Of Cluster Assignments" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://localhost:4000/publications/2017_bmc_clustering/" />
  <meta property="og:description" content=" Background: With ever-increasing amounts of data produced in biology research, scientists are in need of efficient data analysis methods. Cluster analysis, combined with visualization of the results, is one such method that can be used to make sense of large data volumes. At the same time, cluster analysis is known to be imperfect and depends on the choice of algorithms, parameters, and distance measures. Most clustering algorithms don't properly account for ambiguity in the source data, as records are often assigned to discrete clusters, even if an assignment is unclear. While there are metrics and visualization techniques that allow analysts to compare clusterings or to judge cluster quality, there is no comprehensive method that allows analysts to evaluate, compare, and refine cluster assignments based on the source data, derived scores, and contextual data.  Results: In this paper, we introduce a method that explicitly visualizes the quality of cluster assignments, allows comparisons of clustering results and enables analysts to manually curate and refine cluster assignments. Our methods are applicable to matrix data clustered with partitional, hierarchical, and fuzzy clustering algorithms. Furthermore, we enable analysts to explore clustering results in context of other data, for example, to observe whether a clustering of genomic data results in a meaningful differentiation in phenotypes. Conclusions: Our methods are integrated into Caleydo StratomeX, a popular, web-based, disease subtype analysis tool. We show in a usage scenario that our approach can reveal ambiguities in cluster assignments and produce improved clusterings that better differentiate genotypes and phenotypes." />
  
  <meta property="og:image" content="http://localhost:4000/assets/images/papers/2017_bmc_clustering_teaser.png" />
  


</head>


    <body>
    <div class="wrap">
      <div class="navigation">
    <div class="logo">
        <a href="/">
            <!-- JKU Visual Data Science Group -->
            <img class="logo-header-svg" src="/assets/images/logos/DataVisGroup.png" alt="JKU Visual Data Science Group - Logo">
        </a>
    </div>

    <div class="navigation-buttons"> 
        <ul>
            <li class="home-button">
                <a href="/"><span class="nav-icon"><i class="fa fa-home"></i></span><span
                        class="nav-text">Home</span></a>
            </li>
            <li class="team-button">
                <a href="/team/"><span class="nav-icon"><i class="fa fa-user"></i></span><span
                        class="nav-text">Team</span></a>
            </li>
            <li class="publication-button">
                <a href="/publications/"><span class="nav-icon"><i class="fa fa-book"></i></span><span
                        class="nav-text">Publications</span></a>
            </li>
            
            <!-- <li class="project-button">
                <a href="/tools/"><span class="nav-icon"><i class="fa fa-cog"></i></span><span
                        class="nav-text">Tools</span></a>
            </li> -->
            <!--<li class="download-button">
              <a href="/software/"><span class="nav-icon"><i class="fa fa-download"></i></span><span class="nav-text">Software</span></a>
            </li> -->

            <!--<li class="help-button">
              <a href="http://help.caleydo.org"><span class="nav-icon"><i class="fa fa-question-circle"></i></span><span class="nav-text">Help</span></a>
            </li> -->
            <!--<li class="help-button">
                  <a href="/documentation/"><span class="nav-icon"><i class="fa fa-book"></i></span><span class="nav-text">Docs</span></a>
              </li>-->

        </ul>
    </div>

    <div class="logo-subtitle">Part of the <a href="https://www.jku.at/en/institute-of-computer-graphics/">Institute of Computer Graphics</a> @ <a href="https://www.jku.at/">Johannes Kepler University Linz</a>, Austria</div>
</div>


      <div class="publication">

  <header class="publication-title">
    <h1>Interactive Visual Exploration And Refinement Of Cluster Assignments</h1>
  </header>

  

  


  <article class="publication-content-main">

    
    <img src="/assets/images/papers/2017_bmc_clustering_teaser.png" alt="Interactive Clustering Teaser">
    



    <h1>Abstract</h1>
     <p>Background: With ever-increasing amounts of data produced in biology research, scientists are in need of efficient data analysis methods. Cluster analysis, combined with visualization of the results, is one such method that can be used to make sense of large data volumes. At the same time, cluster analysis is known to be imperfect and depends on the choice of algorithms, parameters, and distance measures. Most clustering algorithms don't properly account for ambiguity in the source data, as records are often assigned to discrete clusters, even if an assignment is unclear. While there are metrics and visualization techniques that allow analysts to compare clusterings or to judge cluster quality, there is no comprehensive method that allows analysts to evaluate, compare, and refine cluster assignments based on the source data, derived scores, and contextual data. </p> <p>Results: In this paper, we introduce a method that explicitly visualizes the quality of cluster assignments, allows comparisons of clustering results and enables analysts to manually curate and refine cluster assignments. Our methods are applicable to matrix data clustered with partitional, hierarchical, and fuzzy clustering algorithms. Furthermore, we enable analysts to explore clustering results in context of other data, for example, to observe whether a clustering of genomic data results in a meaningful differentiation in phenotypes.</p> <p>Conclusions: Our methods are integrated into Caleydo StratomeX, a popular, web-based, disease subtype analysis tool. We show in a usage scenario that our approach can reveal ambiguities in cluster assignments and produce improved clusterings that better differentiate genotypes and phenotypes.</p>



      <h1>Citation</h1>
    <div class="citation">
      <p>
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        kern, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
              
                  <a href="/persons/lex/">Alexander Lex</a>, 
              
              <!-- <a href="http://alexander-lex.net">Alexander&nbsp;Lex</a>,  -->
              
            
        
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
              
                  <a href="/persons/gehlenborg/">Nils Gehlenborg</a>, 
              
              <!-- <a href="http://www.gehlenborg.com">Nils&nbsp;Gehlenborg</a>,  -->
              
            
        
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Chris R. Johnson
        
        
        


        <br/>
        <span class="paper-title">
        
          <b><a href="http://data.caleydo.org/papers/2017_bmc_clustering.pdf">Interactive Visual Exploration And Refinement Of Cluster Assignments</a></b>
          
          </span><br/>
        

        <span class="paper-citation">
        
          
          



          BMC Bioinformatics,  18(1): 406,   <a href="https://dx.doi.org/10.1186/s12859-017-1813-7">doi:10.1186/s12859-017-1813-7</a>, 2017.
        

      

      
      </span>
    </div>

      <h1>BibTeX</h1>

      <!-- here replacing the curly brackets with html special characters. { is &#123; ; } is &#125; -->
<pre class="bibtex">
@article&#123;2017_bmc_clustering,
  title = &#123;Interactive Visual Exploration And Refinement Of Cluster Assignments&#125;,
  author = &#123;kern and Alexander Lex and Nils Gehlenborg and Chris R. Johnson&#125;,
  journal = &#123;BMC Bioinformatics&#125;,
  doi = &#123;10.1186/s12859-017-1813-7&#125;,
  url = &#123;http://biorxiv.org/content/early/2017/04/04/123844&#125;,
  volume = &#123;18&#125;,
  number = &#123;1&#125;,
  pages = &#123;406&#125;,
  month = &#123;apr&#125;,
  year = &#123;2017&#125;
}
</pre>


    <!-- main markdown content -->
    <h1 id="acknowledgements">Acknowledgements</h1>

<p>We thank Samuel Gratzl for his help with the implementation. This work was funded in part by the US National Institutes of Health (U01 CA198935, P41 GM103545-17, R00 HG007583) and supported by a fellowship of the FITweltweit program of the German Academic Exchange Service (DAAD).</p>

  </article>

  <article class="publication-content-aside">
    

    


    <h1>Resources</h1>
    <ul>
    

    
    <li>
      <i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2017_bmc_clustering.pdf">Paper (PDF)</a>
    </li>
    
    
    

    
    <li>
      <i class="fa  fa-github"></i> &nbsp;<a href="https://github.com/K3rn1n4tor/stratomex_js">Source Code</a>
    </li>
    
    </ul>

    
    <h1>Videos</h1>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    <div class="video">
      <p><b>Interactive Visual Exploration And Refinement Of Cluster Assignments</b></p>
      
      <div class="video-wrapper">
        <iframe src="//www.youtube.com/embed/gI16xhF9evY?controls=2&showinfo=0&iv_load_policy=3" frameborder="0" allowfullscreen></iframe>
      </div>
      
      
      <p><i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2017_bmc_clustering.mp4">Download Video</a><p/>
      
    </div>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    

  </article>


  <article class="publication-meta">

  </article>

  <article class="publication-resources">
  </article>
</div>


      <footer>
  <div class="footer-content">
    <div class="footer-list-social">
      <ul>
          
          <li>
            <a href="mailto:contact@caleydo.org">
              <i class="fa fa-lg fa-envelope-o"></i>
              <span class="footer-text">Email</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://facebook.com/pages/Caleydo/110550618994500">
              <i class="fa fa-lg fa-facebook"></i>
              <span class="footer-text">Facebook</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/caleydo_org">
              <i class="fa fa-lg fa-twitter"></i>
              <span class="footer-text">Twitter</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://www.youtube.com/user/caleydotugraz">
              <i class="fa fa-lg fa-youtube"></i>
              <span class="footer-text">YouTube</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://github.com/caleydo">
              <i class="fa fa-lg fa-github"></i>
              <span class="footer-text">GitHub</span>
            </a>
          </li>
          
      </ul>
    </div>
  </div>

  <div class="footer-content">
    <!-- <div class="footer-disclaimer">
      <p>
        Caleydo is a joint project of
      </p>
    </div> -->
    <div class="footer-list-institutions">
      <ul>
        
          <li>
            <a href="https://www.jku.at/en/institute-of-computer-graphics/"><img class="institution-logo" src="/assets/images/logos/icg-logo.png" alt="https://www.jku.at/en/institute-of-computer-graphics/"></a>
          </li>
        
          <li>
            <a href="https://www.jku.at"><img class="institution-logo" src="/assets/images/logos/johannes-kepler-university-linz.png" alt="https://www.jku.at"></a>
          </li>
        
      </ul>
    </div>
 </div>


  <div class="footer-content">
    <!-- <span class="footer-logo">
      <a href="/">
        <img class="logo-footer-svg" src="/assets/images/logos/caleydo_c.svg" alt="Caleydo Logo">
      </a>
    </span> -->
    <div class="footer-disclaimer">
      <p>
        Copyright &copy; 2019 The Caleydo Team. All content on this website is licensed under the <br />
        <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons Attribution license (CC BY)</a>. Copyright of the linked papers is with the publishers.
      </p>
    </div>

  </div>

</footer>

    </div>

    
    <!-- Google Analytics -->
<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-45998043-1', 'auto');
    ga('send', 'pageview');



    (function (){
        var links = document.getElementsByTagName('a');
        var myLink = null;
        var startWith = null;

        for(var i = 0; i < links.length; ++i) {
            myLink = links[i];

            startWith = myLink.getAttribute('href').slice(0, 1);

            if (startWith!="/" && startWith!="#"){
                addListener(myLink,'mousedown',function(){

                    console.log("a:", this.getAttribute('href'))
                    ga('send', 'event', 'link', 'click', this.getAttribute('href'));

                })
            }

        }
    })();



    /**
     * Utility to wrap the different behaviors between W3C-compliant browsers
     * and IE when adding event handlers.
     *
     * @param {Object} element Object on which to attach the event listener.
     * @param {string} type A string representing the event type to listen for
     *     (e.g. load, click, etc.).
     * @param {function()} callback The function that receives the notification.
     */
    function addListener(element, type, callback) {
        if (element.addEventListener) element.addEventListener(type, callback);
        else if (element.attachEvent) element.attachEvent('on' + type, callback);
    }




</script>
<!-- End Google Analytics -->
     
    </body>
</html>