<!DOCTYPE html>
<html>

  <head>
 <meta charset="utf-8">
<title>Vials: Visualizing Alternative Splicing of Genes</title>
<meta name="viewport" content="width=device-width">
<meta name="description" content="Caleydo: Visualization for BioMedicine.">
<link rel="canonical" href="http://localhost:4000/publications/2015_infovis_vials/">

<!-- Custom Fonts -->
<link href='//fonts.googleapis.com/css?family=Lato:300,400,700,900,300italic,400italic,700italic,900italic|Patua+One|Vollkorn:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
<link href='//fonts.googleapis.com/css?family=PT+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>

<!-- Custom CSS -->
<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
<link rel="stylesheet" href="/css/site.css">

<!-- Fav and touch icons -->
<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
<link rel="apple-touch-icon" href="/favicon.png" />

 <!-- metadata for link preview  -->
  <!-- Schema.org markup for Google+ -->
  <meta itemprop="name" content="Vials: Visualizing Alternative Splicing of Genes">
  <meta itemprop="description" content=" Alternative splicing is a process by which the same DNA sequence is used to assemble different proteins, called protein isoforms. Alternative splicing works by selectively omitting some of the coding regions (exons) typically associated with a gene. Detection of alternative splicing is difficult and uses a combination of advanced data acquisition methods and statistical inference. Knowledge about the abundance of isoforms is important for understanding both normal processes and diseases and to eventually improve treatment through targeted therapies. The data, however, is complex and current visualizations for isoforms are neither perceptually efficient nor scalable. To remedy this, we developed Vials, a novel visual analysis tool that enables analysts to explore the various datasets that scientists use to make judgments about isoforms: the abundance of reads associated with the coding regions of the gene, evidence for junctions, i.e., edges connecting the coding regions, and predictions of isoform frequencies. Vials is scalable as it allows for the simultaneous analysis of many samples in multiple groups. Our tool thus enables experts to (a) identify patterns of isoform abundance in groups of samples and (b) evaluate the quality of the data. We demonstrate the value of our tool in case studies using publicly available datasets.">
  
  <meta itemprop="image" content="http://localhost:4000/assets/images/papers/2015_infovis_vials_teaser.png">
  

  <!-- Twitter Card data -->
  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@caleydo_org">
  <meta name="twitter:title" content="Vials: Visualizing Alternative Splicing of Genes">
  <meta name="twitter:description" content=" Alternative splicing is a process by which the same DNA sequence is used to assemble different proteins, called protein isoforms. Alternative splicing works by selectively omitting some of the coding regions (exons) typically associated with a gene. Detection of alternative splicing is difficult and uses a combination of advanced data acquisition methods and statistical inference. Knowledge about the abundance of isoforms is important for understanding both normal processes and diseases and to eventually improve treatment through targeted therapies. The data, however, is complex and current visualizations for isoforms are neither perceptually efficient nor scalable. To remedy this, we developed Vials, a novel visual analysis tool that enables analysts to explore the various datasets that scientists use to make judgments about isoforms: the abundance of reads associated with the coding regions of the gene, evidence for junctions, i.e., edges connecting the coding regions, and predictions of isoform frequencies. Vials is scalable as it allows for the simultaneous analysis of many samples in multiple groups. Our tool thus enables experts to (a) identify patterns of isoform abundance in groups of samples and (b) evaluate the quality of the data. We demonstrate the value of our tool in case studies using publicly available datasets.">
  <meta name="twitter:creator" content="@caleydo_org">
  
  <meta name="twitter:image:src" content="http://localhost:4000/assets/images/papers/2015_infovis_vials_teaser.png">
  

  <!-- Open Graph data -->
  <meta property="og:title" content="Vials: Visualizing Alternative Splicing of Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://localhost:4000/publications/2015_infovis_vials/" />
  <meta property="og:description" content=" Alternative splicing is a process by which the same DNA sequence is used to assemble different proteins, called protein isoforms. Alternative splicing works by selectively omitting some of the coding regions (exons) typically associated with a gene. Detection of alternative splicing is difficult and uses a combination of advanced data acquisition methods and statistical inference. Knowledge about the abundance of isoforms is important for understanding both normal processes and diseases and to eventually improve treatment through targeted therapies. The data, however, is complex and current visualizations for isoforms are neither perceptually efficient nor scalable. To remedy this, we developed Vials, a novel visual analysis tool that enables analysts to explore the various datasets that scientists use to make judgments about isoforms: the abundance of reads associated with the coding regions of the gene, evidence for junctions, i.e., edges connecting the coding regions, and predictions of isoform frequencies. Vials is scalable as it allows for the simultaneous analysis of many samples in multiple groups. Our tool thus enables experts to (a) identify patterns of isoform abundance in groups of samples and (b) evaluate the quality of the data. We demonstrate the value of our tool in case studies using publicly available datasets." />
  
  <meta property="og:image" content="http://localhost:4000/assets/images/papers/2015_infovis_vials_teaser.png" />
  


</head>


    <body>
    <div class="wrap">
      <div class="navigation">
    <div class="logo">
        <a href="/">
            <!-- JKU Visual Data Science Group -->
            <img class="logo-header-svg" src="/assets/images/logos/DataVisGroup.png" alt="JKU Visual Data Science Group - Logo">
        </a>
    </div>

    <div class="logo-subtitle">Part of the <a href="https://www.jku.at/en/institute-of-computer-graphics/" target="_blank">Institute of Computer Graphics</a> @ <a href="https://www.jku.at/" target="_blank">Johannes Kepler University Linz</a>, Austria</div>


    <div class="navigation-buttons"> 
        <ul>
            <li class="home-button">
                <a href="/"><span class="nav-icon"><i class="fa fa-home"></i></span><span
                        class="nav-text">Home</span></a>
            </li>
            <li class="team-button">
                <a href="/team/"><span class="nav-icon"><i class="fa fa-user"></i></span><span
                        class="nav-text">Team</span></a>
            </li>
            <li class="publication-button">
                <a href="/publications/"><span class="nav-icon"><i class="fa fa-book"></i></span><span
                        class="nav-text">Publications</span></a>
            </li>
            <li class="teaching-button">
                <a href="https://www.jku.at/en/institute-of-computer-graphics/teaching/" target="_blank"><span class="nav-icon"><i class="fa fa-group"></i></span><span
                        class="nav-text">Teaching</span></a>
            </li>
            
            <!-- <li class="project-button">
                <a href="/tools/"><span class="nav-icon"><i class="fa fa-cog"></i></span><span
                        class="nav-text">Tools</span></a>
            </li> -->
            <!--<li class="download-button">
              <a href="/software/"><span class="nav-icon"><i class="fa fa-download"></i></span><span class="nav-text">Software</span></a>
            </li> -->

            <!--<li class="help-button">
              <a href="http://help.caleydo.org"><span class="nav-icon"><i class="fa fa-question-circle"></i></span><span class="nav-text">Help</span></a>
            </li> -->
            <!--<li class="help-button">
                  <a href="/documentation/"><span class="nav-icon"><i class="fa fa-book"></i></span><span class="nav-text">Docs</span></a>
              </li>-->

        </ul>
    </div>

    <!-- <div class="logo-subtitle">Part of the <a href="https://www.jku.at/en/institute-of-computer-graphics/" target="_blank">Institute of Computer Graphics</a> @ <a href="https://www.jku.at/" target="_blank">Johannes Kepler University Linz</a>, Austria</div> -->
</div>


      <div class="publication">

  <header class="publication-title">
    <h1>Vials: Visualizing Alternative Splicing of Genes</h1>
  </header>

  

  


  <article class="publication-content-main">

    
    <img src="/assets/images/papers/2015_infovis_vials_teaser.png" alt="Vials Teaser">
    


    
    <h1>Abstract</h1>
     <p>Alternative splicing is a process by which the same DNA sequence is used to assemble different proteins, called protein isoforms. Alternative splicing works by selectively omitting some of the coding regions (exons) typically associated with a gene. Detection of alternative splicing is difficult and uses a combination of advanced data acquisition methods and statistical inference. Knowledge about the abundance of isoforms is important for understanding both normal processes and diseases and to eventually improve treatment through targeted therapies. The data, however, is complex and current visualizations for isoforms are neither perceptually efficient nor scalable. To remedy this, we developed Vials, a novel visual analysis tool that enables analysts to explore the various datasets that scientists use to make judgments about isoforms: the abundance of reads associated with the coding regions of the gene, evidence for junctions, i.e., edges connecting the coding regions, and predictions of isoform frequencies. Vials is scalable as it allows for the simultaneous analysis of many samples in multiple groups. Our tool thus enables experts to (a) identify patterns of isoform abundance in groups of samples and (b) evaluate the quality of the data. We demonstrate the value of our tool in case studies using publicly available datasets.</p>
    <br>
    

    <h1>Citation</h1>
    <div class="citation">
      <p>
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Hendrik Strobelt, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Bilal Alsallakh, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Joseph Botors, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Brant Peterson, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Mark Borowsky, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Hanspeter Pfister, 
        
        
        
          
            
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
              
        
        
        Alexander Lex
        
        
        


        <br/>
        <span class="paper-title">
          
            <b><a href="http://data.caleydo.org/papers/2015_infovis_vials.pdf">Vials: Visualizing Alternative Splicing of Genes</a></b>
          
        </span><br/>
        

        <span class="paper-citation">
        
          
          



          IEEE Transactions on Visualization and Computer Graphics (InfoVis '15),  22(1): 399--408,   <a href="https://dx.doi.org/10.1109/TVCG.2015.2467911">doi:10.1109/TVCG.2015.2467911</a>, 2016.
        

      

      
      </span>
    </div>

    
      <h1>BibTeX</h1>
      
            <!-- here replacing the curly brackets with html special characters. { is &#123; ; } is &#125; -->
      <pre class="bibtex">
      @article&#123;2015_infovis_vials,
        title = &#123;Vials: Visualizing Alternative Splicing of Genes&#125;,
        author = &#123;Hendrik Strobelt and Bilal Alsallakh and Joseph Botors and Brant Peterson and Mark Borowsky and Hanspeter Pfister and Alexander Lex&#125;,
        journal = &#123;IEEE Transactions on Visualization and Computer Graphics (InfoVis '15)&#125;,
        doi = &#123;10.1109/TVCG.2015.2467911&#125;,
        volume = &#123;22&#125;,
        number = &#123;1&#125;,
        pages = &#123;399--408&#125;,
        year = &#123;2016&#125;
      }
      </pre>
    

    <!-- main markdown content -->
    <h1 id="acknowledgements">Acknowledgements</h1>

<p>We thank Samuel Gratzl for help with the framework, Nils Gehlenborg for comments on the manuscript, and Sebastian Hörsch and other members of the SDA team for their expertise and feedback. This work was supported in part by Novartis Institutes for BioMedical Research, the Austrian Science Fund (J 3437-N15), the Air Force Research Laboratory and DARPA grant FA8750-12-C-0300, and the US National Institutes of Health (U01 CA198935).</p>

  </article>

  <article class="publication-content-aside">
    

    
    <h1>Tool Website</h1>
    <ul class="project-icon-list">
            <li><a href="http://vials.io">Vials Website</a></li>
    </ul>
    


    <h1>Resources</h1>
    <ul>
    
    <li>
      <i class="fa fa-globe"></i> &nbsp;<a href="https://dx.doi.org/10.1109/TVCG.2015.2467911">Publisher</a>
    </li>
    

    

    
    <li>
      <i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2015_infovis_vials.pdf">Paper (PDF)</a>
    </li>
    
    
    <li>
      <i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2015_infovis_vials_supplement.pdf">Supplementary Material</a>
    </li>
    
    
	
      <li>
        
        <i class="fa fa-external-link"></i>
        
        &nbsp;<a href="http://vials.io/talk/">Vials Conference Talk Slides</a>
      </li>
	
    

    
    <li>
      <i class="fa  fa-github"></i> &nbsp;<a href="https://github.com/caleydo/vials/">Source Code</a>
    </li>
    
    </ul>

    
    <h1>Videos</h1>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    <div class="video">
      <p><b>Vials Demonstration Video</b></p>
      
      <div class="video-wrapper">
        <iframe src="//www.youtube.com/embed/Nr3E61zq-OY?controls=2&showinfo=0&iv_load_policy=3" frameborder="0" allowfullscreen></iframe>
      </div>
      
      
      <p><i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2015_infovis_vials.mp4">Download Video</a><p/>
      
    </div>
    
    
    
    <div class="video">
      <p><b>Vials Preview Video</b></p>
      
      <div class="video-wrapper">
        <iframe src="//www.youtube.com/embed/zsx0Uii4mc4?controls=2&showinfo=0&iv_load_policy=3" frameborder="0" allowfullscreen></iframe>
      </div>
      
      
      <p><i class="fa fa-download"></i> &nbsp;<a href="http://data.caleydo.org/papers/2015_infovis_vials_preview.mp4">Download Video</a><p/>
      
    </div>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    

  </article>


  <article class="publication-meta">

  </article>

  <article class="publication-resources">
  </article>
</div>


      <footer>
  <div class="footer-content">
    <div class="footer-list-social">
      <ul>
          
          <li>
            <a href="mailto:contact@caleydo.org">
              <i class="fa fa-lg fa-envelope-o"></i>
              <span class="footer-text">Email</span>
            </a>
          </li>
          

          <!--  -->

          
          <li>
            <a href="https://twitter.com/marc_streit">
              <i class="fa fa-lg fa-twitter"></i>
              <span class="footer-text">Twitter</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://www.youtube.com/user/caleydotugraz">
              <i class="fa fa-lg fa-youtube"></i>
              <span class="footer-text">YouTube</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://github.com/caleydo">
              <i class="fa fa-lg fa-github"></i>
              <span class="footer-text">GitHub</span>
            </a>
          </li>
          
      </ul>
    </div>
  </div>

  <div class="footer-content">
    <!-- <div class="footer-disclaimer">
      <p>
        Caleydo is a joint project of
      </p>
    </div> -->
    <div class="footer-list-institutions">
      <ul>
        
          <li>
            <a href="https://www.jku.at/en/institute-of-computer-graphics/"><img class="institution-logo" src="/assets/images/logos/icg-logo.png" alt="https://www.jku.at/en/institute-of-computer-graphics/"></a>
          </li>
        
          <li>
            <a href="https://www.jku.at"><img class="institution-logo" src="/assets/images/logos/johannes-kepler-university-linz.png" alt="https://www.jku.at"></a>
          </li>
        
      </ul>
    </div>
 </div>


  <div class="footer-content">
    <!-- <span class="footer-logo">
      <a href="/">
        <img class="logo-footer-svg" src="/assets/images/logos/caleydo_c.svg" alt="Caleydo Logo">
      </a>
    </span> -->
    <div class="footer-disclaimer">
      <p>
        Copyright &copy; 2020 JKU Visual Data Science Group. All content on this website is licensed under the <br />
        <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons Attribution license (CC BY)</a>. Copyright of the linked papers is with the publishers.
      </p>
    </div>

  </div>

</footer>

    </div>

    
    <!-- Google Analytics -->
<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-45998043-1', 'auto');
    ga('send', 'pageview');



    (function (){
        var links = document.getElementsByTagName('a');
        var myLink = null;
        var startWith = null;

        for(var i = 0; i < links.length; ++i) {
            myLink = links[i];

            startWith = myLink.getAttribute('href').slice(0, 1);

            if (startWith!="/" && startWith!="#"){
                addListener(myLink,'mousedown',function(){

                    console.log("a:", this.getAttribute('href'))
                    ga('send', 'event', 'link', 'click', this.getAttribute('href'));

                })
            }

        }
    })();



    /**
     * Utility to wrap the different behaviors between W3C-compliant browsers
     * and IE when adding event handlers.
     *
     * @param {Object} element Object on which to attach the event listener.
     * @param {string} type A string representing the event type to listen for
     *     (e.g. load, click, etc.).
     * @param {function()} callback The function that receives the notification.
     */
    function addListener(element, type, callback) {
        if (element.addEventListener) element.addEventListener(type, callback);
        else if (element.attachEvent) element.attachEvent('on' + type, callback);
    }




</script>
<!-- End Google Analytics -->
     
    </body>
</html>