---
layout: component
title: StratomeX
key: stratomex
permalink: /components/stratomex/
status: ongoing
standfirst: A tool for integrative visualization of cancer genomics data from The Cancer Genome Atlas project to identify and characterize tumor subtypes.
website: http://stratomex.caleydo.org
publications:
 - 2014-nmc-stratomex
 - 2012-eurovis-stratomex 
collaborators:
 - gehlenborg
 - streit 
---

Identification and characterization of cancer subtypes for refined therapeutic targeting is an important step towards improved patient outcomes. Recent research has shown that integrated analysis of different molecular data types generated by the TCGA project can be used to discover subtypes and suggest molecular differences relevant for therapeutic approaches.

Large-scale data analysis systems developed to perform comprehensive analyses of TCGA data, such as the Firehose pipeline developed by the Broad Institute, have strong algorithmic capabilities and generate analysis results required for subtype identification and characterization. Visualization tools that enable biologists to integrate and explore these large sets of results are an important prerequisite for their efficient interpretation.

To address this, we have developed a new interactive visualization method that supports biologists in the identification and characterization of cancer subtypes. Our method is designed to integrate many TCGA data types, including expression, copy number, methylation, mutation, and clinical data.

The core concept of our approach is to visualize sample groupings (partitions of the set of all samples for a given tumor type) and the relationships between these groupings in a given cancer type. Groupings of samples are obtained for instance by clustering mRNA, microRNA or DNA methylation profiles. Groups can also be derived from copy number levels of a particular gene or gene mutation status, e.g., one group for “wild-type”, one for “mutated”.

The visualization method has been implemented in the Caleydo Visualization Framework.